tests/test_parsers/test_docstrings/test_numpy.py

```diff
@@ -81,6 +81,37 @@ def test_sections_without_signature():
         assert "param" in error
 
 
+def test_sections_without_description():
+    """Parse a docstring without descriptions."""
+    # type of return value always required
+    sections, errors = parse(
+        """
+        Sections without descriptions.
+
+        Parameters
+        ----------
+        void : str
+        niet : str
+        nada : str
+        rien : str
```

Comment:
You can probably use only one parameter, that would be enough.

Reply:
One would probably be ok, four is definitely more than necessary. I'm changing this to two parameters because I want to make sure it works with one as well as multiple. The Exceptions and Returns sections verify the one parameter case. Knowing the implementation, I don't know that this is necessary, but I like to ignore my knowledge of the implementation to some extent when writing tests.

Reply:
Very valid point! Thanks for expressing it :slightly_smiling_face: